cypress/support/commands.js

```diff
@@ -22,4 +22,31 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+Cypress.Commands.add('loginToAuth0', (username, password) => {
+    const log = Cypress.log({
+      displayName: 'AUTH0 LOGIN',
+      message: [`🔐 Authenticating | ${username}`],
+      autoEnd: false,
```

Comment:
still wish we could have gotten auth0 testing to work! darn! but glad we made the progress we did!